components/automatic-grid/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "automatic-grid",
+  "version": "1.0.0",
+  "description": "A component for Coherent Labs Gameface.",
+  "main": "script.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/CoherentLabs/GameUIComponents/tree/master/components/automatic-grid"
+  },
+  "keywords": [
+    "UI",
+    "Component"
+  ],
+  "files": [
+    "index.js",
+    "package.json",
+    "README.md",
+    "script.js",
+    "style.css",
+    "template.html",
+    "demo/",
+    "cjs/",
+    "umd/",
+    "components-theme.css"
```

Comment:
If the files property is omitted all files are packaged. We can skip this list if these are all files.